wallet/src/Pos/Wallet/Web/Methods/Logic.hs

```diff
@@ -115,7 +115,7 @@ getAccountMod
     -> AccountId
     -> m CAccount
 getAccountMod balAndUtxo accMod accId = do
-    dbAddrs    <- getAccountAddrsOrThrow Existing accId
+    dbAddrs    <- getAccountAddrsOrThrow Existing (NeedSorting True) accId
```

Comment:
This is the only one that needs the sorting. In a future version of this patch for other branches, we should simply sort the output here, and not need to pass the `NeedSorting` inward.